client/src/containers/NychaPage.js

```diff
@@ -83,6 +83,8 @@ export default class NychaPage extends Component {
         } : 
         null );
 
+    const takeActionURL = (usersInputAddress && ('https://app.justfix.nyc/ddo?address=' + encodeURIComponent((usersInputAddress.housenumber + ' ' + usersInputAddress.streetname).trim()) + '&borough=' + encodeURIComponent(usersInputAddress.boro) + '/?utm_source=whoownswhat_take_action_nycha'));
```

Comment:
I think you will want the part before the `utm_source` to be a single ampersand (`&`) rather than `/?`.

Reply:
It looks like we actually need neither: https://en.wikipedia.org/wiki/UTM_parameters. Or am I missing something?

Reply:
Hmm, that's not a valid querystring, though... At least from what I can tell, it looks like you will be generating a URL like `https://foo.com/?address=boop&borough=BROOKLYN/?utm_source=whoownswhat_take_action_nycha`, which means that instead of `address`, `borough`, and `utm_source` arguments, a querystring parser will only see `address` and `borough` (the latter will have a value of `BROOKLYN/?utm_source=whoownswhat_take_action_nycha`).  Maybe I am mis-reading your code though...

Reply:
e.g., if you run `node` and give it the following:

```js
require('querystring').parse('address=boop&borough=BROOKLYN/?utm_source=whoownswhat_take_action_nycha')
```

you will get:

```js
{
  address: 'boop',
  borough: 'BROOKLYN/?utm_source=whoownswhat_take_action_nycha'
}
```

Reply:
ohhh I see what you mean. Mentally, I've been treating the `address` and `borough` arguments as part of the "URL" and the `utm_source` as an additional argument, when they all should be treated equally as additional arguments.